README.md

```diff
@@ -1,2 +1,82 @@
 # packer-workflows
 A packer project to build an AMI containing software for running bioinformatics workflows
+
+## Development
+
+### Setup
+* Install [packer](https://www.packer.io/intro/getting-started/install.html)
+* Install [ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
+
+### Validate a template
+Choose an ImageName such as "MyImage" and run
+```
+packer validate -var 'ImageName=MyImage' template.json
+```
+
+#### Access
+* Request an IAM account in [Imagecentral](https://github.com/Sage-Bionetworks/imagecentral-infra)
+* Change password and set up MFA
+* Create a Keypair
+* Add your access code and secrety key to `~/.aws/credentials`, using a profile such as "imagecentral.jsmith"
+* Authenticate with `awsmfa`, for example `awsmfa -i imagecentral.jsmith -t jsmith@imagecentral`
+* Finally, add the following
+```
+[profile packer-service-imagecentral]
+region = us-east-1
+role_arn = arn:aws:iam::867686887310:role/accounts-PackerServiceRole-18QJ34FLREC6E
```

Comment:
might be good to not provide the real role_arn?

Reply:
fixed and overwrote commit